docs/downloads.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Downloads"
+permalink: /downloads/
+---
+
+<h2>Git repos</h2>
+On going development is done on <a href="https://github.com/shadow-maint/shadow">github</a>.
```

Comment:
I think that it's spelled "ongoing" but I'm not a native English speaker so I'm not completely sure.

Note: https://english.stackexchange.com/questions/9240/ongoing-or-on-going seems to agree.